articles/native-platforms/ionic/05-rules.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Rules
+description: This tutorial will show you how to use Auth0 rules to extend what Auth0 has to offer.
+---
+
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '05-Rules',
+  pkgFilePath: null,
+  pkgType: 'replace'
+}) %>
+
+<%= include('../../_includes/server-platform/_rules-introduction') %>
```

Comment:
@lobo Throws `Error: Could not find include ../../_includes/server-platform/_rules-introduction`. There is no such directory.